src/components/views/dialogs/MultiInviteDialog.js

```diff
@@ -41,20 +42,28 @@ export default class MultiInviteDialog extends React.Component {
         }
     }
 
+    componentWillUnmount() {
+        this._unmounted = true;
```

Comment:
Point - it should definitely cancel if you dismiss the dialog so I've just made them the same variable.